### PR TITLE
Bug 2028685: LSO repeatedly reports errors while diskmaker-discovery pod is starting

### PR DESCRIPTION
**Pull request by RomanBednar**
PR contains two commits, one for documenting how to run LSO locally which can be useful for verification of some bug fixes (like this one) and second with an actual fix.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -6,6 +6,42 @@
...
+
+6. Run the operator locally
+```
+~> ./_output/bin/local-storage-operator -kubeconfig=$KUBECONFIG
+```
+
+**NOTE** 
+The diskmaker-discovery pods fail to start until [BZ2027008](https://bugzilla.redhat.com/show_bug.cgi?id=2027008) is fixed
+
+
```
Comment (dobsonj) [inline HACKING.md:44]: Can you remove this note? Notes about specific bugs will (hopefully) quickly become obsolete.

File: HACKING.md [hunk 2]
```diff
@@ -6,6 +6,42 @@
...
+## Running the operator locally
+
+1. Install LSO via OLM/OperatorHub in GUI
+
+2. Build the operator
+```
+make build-operator
+```
+
+3. Export env variables required by LSO [deployment](https://github.com/openshift/local-storage-operator/blob/master/config/manifests/4.10/local-storage-operator.clusterserviceversion.yaml#L363)
```
Comment (dobsonj) [inline HACKING.md:18]: Can you make this a permalink, so the link doesn't get broken when we move these to 4.11?

**Review by dobsonj — COMMENTED** [review]
I'm okay with the fix, but couple of suggestions on the notes.

File: HACKING.md (hunk 1 above)
Comment (dobsonj) [reply HACKING.md:44]: Filed a PR at https://github.com/openshift/local-storage-operator/pull/301

Comment (RomanBednar) [reply HACKING.md]: Done, thanks for the PR.

File: HACKING.md (hunk 2 above)
Comment (RomanBednar) [reply HACKING.md]: Good point - changed.

**Comment by dobsonj** [comment]
Looks good, thanks for the changes. Now we just need https://github.com/openshift/local-storage-operator/pull/301 to merge so e2e-operator will pass.

**Pull request closed without merging**